### DISCUSSION TITLE
Working trying to serialize Apex Class on C#

### DISCUSSION THREAD

**Discussion by jhony251** [dangmai/apex-ast-serializer#83] [General]
Found you works has been a greatfull sorprise.
I would like to use your work on my own job. I have been cheking for MIT licenseand i notice that is very permisive, but i think is very important to ask the owner too.
please let me know if you are agree.

**Comment by dangmai** [comment]
Yes feel free to use or modify this library in whichever way you want, I'm glad to hear that you find it useful.

Please note that the main use for this library is to be used in Prettier Apex, so I can't guarantee API stability for it in the future. If I need to change something in this library there likely won't be any notice or deprecation.
